### DISCUSSION TITLE
Add view method with "List" like mobile verion

### DISCUSSION THREAD

**Discussion by grxgr-hk** [jellyfin/jellyfin-androidtv#4133] [General]
Hello, dev
I installed the Android TV version 0.17.18 on my android projector. However, it is quite inconvenient to explore my library.
There are only 3 show methods (image type) in folders, Poster, thumb & banner. Is it possible to add "List"  to show the full file name in the next version? Just like the mobile version.

Million thanks!

**Comment by monitorjbl** [comment]
+1 to this. An alphabet slider would also be nice for both views. The Plex app on Android TV has this and it makes scrolling a lot less painful for big libraries.

**Reply by tom9123** [reply]
I also prefer this list view feature, the other ones confuse me with the big pictures

**Comment by monitorjbl** [comment]
The mobile app has this too:

![Screenshot_20241217-173327](https://github.com/user-attachments/assets/078e94a9-aa49-4a35-b558-6e8d43ea8611)

It would go a *long* way towards making the Android TV app more usable. Having to always search with text/voice is useful when you know precisely where you want to go, but it's not great for browsing.
